### PR TITLE
API model updates

### PR DESCRIPTION
**Pull request by pezholio**
Updating hmpps-approved-premises-api models from OpenAPI specification.  This PR was created automatically from the generate-types.yml Workflow

### CODE REVIEW

**Review by bryangaledxw — APPROVED** [review]
Looks good. What might be worth doing though is `import type { TemporaryAccommodationLostBed as LostBed }`, to cut down on some of the verbosity. We're doing that with TemporaryAccommodationPremises

File: server/testutils/factories/lostBed.ts [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
-export default Factory.define<LostBed>(() => ({
+export default Factory.define<TemporaryAccommodationLostBed>(() => ({
   id: faker.datatype.uuid(),
   notes: faker.lorem.sentence(),
   startDate: DateFormats.dateObjToIsoDate(faker.date.soon()),
   endDate: DateFormats.dateObjToIsoDate(faker.date.future()),
   numberOfBeds: faker.datatype.number({ max: 10 }),
   referenceNumber: faker.datatype.uuid(),
   reason: referenceDataFactory.lostBedReasons().build(),
+  status: faker.helpers.arrayElement(['active', 'cancelled'] as const),
```
Comment (bryangaledxw) [inline server/testutils/factories/lostBed.ts]: Also, is the `as const` needed? It seemed to compile fine for me without that

**Comment by patrickjfl** [comment]
> Looks good. What might be worth doing though is `import type { TemporaryAccommodationLostBed as LostBed }`, to cut down on some of the verbosity. We're doing that with TemporaryAccommodationPremises

Yeah, makes sense.

File: server/testutils/factories/lostBed.ts (hunk 1 above)
Comment (patrickjfl) [reply server/testutils/factories/lostBed.ts]: no, you're right - removed

**Pull request merged**